README.md

```diff
@@ -1,2 +1,175 @@
-# Microverseday3pr
-this repository for Microverse day3 project.
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portofolio - day3 project <a name="about-project"></a>
+
+
+
+** Portofolio - day3 project** is a project that create simple portofolio sections using html and css for the learning purpose.
+
+## 🛠 Built With <a name="built-with"></a>
+  -Visual Code Studio
+  <br>
+  -Github
+
+### Tech Stack <a name="tech-stack"></a>
+
+ -HTML <br>
+ -CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Microverse</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github</li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **sigma portofolio template - 2 sections**
+- **CSS styling**
+- **linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> 
+
+- [https://github.com/yashodhicy/Microverseday3pr](https://google.com)
```

Comment:
[OPTIONAL] Kindly replace the link with the live demo of the project or replace it with "Coming Soon" so as not to confuse the users. 🥺